frontend/src/lib/components/Annotations/Annotations.tsx

```diff
@@ -2,19 +2,20 @@ import React from 'react'
 import { annotationsLogic } from './annotationsLogic'
 import { useValues, useActions } from 'kea'
 import { AnnotationMarker } from './AnnotationMarker'
-import { AnnotationType, AnnotationScope } from '~/types'
+import { AnnotationType, AnnotationScope, InsightShortId } from '~/types'
 import { dayjs } from 'lib/dayjs'
 
 interface AnnotationsProps {
     dates: string[]
     leftExtent: number
     interval: number
     topExtent: number
-    insightId?: number
+    insightShortId?: InsightShortId
+    insightId?: number // only used for annotations
```

Comment:
The comment on the second line here is weird... I mean, we're in annotations, so `// only used for annotations`  feels kind of "duh"...? :)